tox.ini

```diff
@@ -14,7 +14,7 @@ setenv =
     dbt{100,130}: PIP_CONSTRAINT=constraints/{envname}.txt
 allowlist_externals =
     make
-pip_pre = true
+pip_pre = false
```

Comment:
Unrelated change discussed in maintainers' Slack channel: Don't install pre-release packages. It destabilizes the build; there's no good reason for doing this AFAIK.